src/main/java/oof/command/CalendarCommand.java

```diff
@@ -0,0 +1,155 @@
+package oof.command;
+
+import java.time.YearMonth;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Comparator;
+
+import oof.Storage;
+import oof.TaskList;
+import oof.Ui;
+import oof.task.Deadline;
+import oof.task.Event;
+import oof.task.Task;
+import oof.task.Todo;
+
+/**
+ * Represents a Command to print calendar.
+ */
+public class CalendarCommand extends Command {
+
+    private int month;
+    private int year;
+    private static final int INDEX_DATE = 0;
+    private static final int INDEX_TIME = 1;
+    private static final int INDEX_DAY = 0;
+    private static final int INDEX_MONTH = 1;
+    private static final int INDEX_YEAR = 2;
+    private static final int MONTH_JANUARY = 1;
+    private static final int MONTH_DECEMBER = 12;
+    private static final int SIZE_CALENDAR = 32;
+    private ArrayList<ArrayList<String[]>> calendarTasks = new ArrayList<>(SIZE_CALENDAR);
+
+    /**
+     * Constructor for CalendarCommand.
+     *
+     * @param argumentArray Array of command arguments input by user.
+     */
+    public CalendarCommand(String[] argumentArray) {
+        while (calendarTasks.size() != SIZE_CALENDAR) {
+            calendarTasks.add(new ArrayList<>());
+        }
+        Calendar calendar = Calendar.getInstance();
+        try {
+            this.month = Integer.parseInt(argumentArray[INDEX_MONTH]);
+            this.year = Integer.parseInt(argumentArray[INDEX_YEAR]);
+            if (month < MONTH_JANUARY || month > MONTH_DECEMBER) {
+                // 0-based indexing to 1-based indexing
+                this.month = calendar.get(Calendar.MONTH) + 1;
+                this.year = calendar.get(Calendar.YEAR);
+            }
```

Comment:
I (as the reader) don't get what this code is trying to achieve. 

I.e.: Write a comment for non-trivial codes